uber/config.py

```diff
@@ -175,10 +197,6 @@ def PREREG_AGE_GROUP_OPTS(self):
         return [opt for opt in self.AGE_GROUP_OPTS if opt[0] != self.AGE_UNKNOWN]
 
     @property
-    def DISPLAY_ONEDAY_BADGES(self):
-        return self.ONE_DAYS_ENABLED and sa.days_before(30, self.EPOCH)
```

Comment:
This option was actually never used; we put it in last year expecting that we'd turn on single day badge purchases a month before the event, but that was never implemented, so I removed the property.